plugins/crust-gather.yaml

```diff
@@ -0,0 +1,48 @@
+# This file was generated by GoReleaser. DO NOT EDIT.
+apiVersion: krew.googlecontainertools.github.com/v1alpha2
+kind: Plugin
+metadata:
+  name: crust-gather
+spec:
+  version: v0.5.3
+  homepage: https://github.com/crust-gather/crust-gather
+  platforms:
+  - bin: kubectl-crust-gather.exe
+    uri: https://github.com/crust-gather/crust-gather/releases/download/v0.5.3/kubectl-crust-gather_v0.5.3_windows_amd64.tar.gz
+    sha256: 6c3e3697ad33ad190fd2398776fc9a2b28288147c57991878c9d48ed38063f23
+    selector:
+      matchLabels:
+        os: windows
+        arch: amd64
```

Comment:
consider releasing for windows/arm64 later (doesn't have to be in this PR).

Reply:
I’ll have a look into this, thanks